.github/workflows/release.yml

```diff
@@ -105,6 +43,18 @@ jobs:
           cp target/${{ matrix.target }}/release/git-remote-rad "$staging"/
           tar czf "$staging.tar.gz" "$staging"
           cp "$staging.tar.gz" "$staging"/
+      - shell: bash
+        run: 'echo "$SSH_PRIVATE_KEY" >ssh-private-key'
+        env:
+          SSH_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
+      - shell: bash
+        run: chmod go-rwx ssh-private-key
+      - shell: bash
+        run: ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ssh-private-key github-actions@files.radicle.xyz mkdir -p "heartwood-artifacts/${{ matrix.target }}/${{ github.sha }}"
+      - shell: bash
+        run: scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ssh-private-key -r "${{ matrix.target }}/*" "github-actions@files.radicle.xyz:heartwood-artifacts/${{ matrix.target }}/${{ github.sha }}/"
+      - shell: bash
+        run: ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ssh-private-key github-actions@files.radicle.xyz ln -sf "heartwood-artifacts/${{ matrix.target }}/${{ github.sha }}" "heartwood-artifacts/${{ matrix.target }}/latest"
```

Comment:
Can we `s/heartwood-artifacts/releases`?

Reply:
Yep